diff -up ffmpeg2theora-0.26/SConstruct.ldflags ffmpeg2theora-0.26/SConstruct
--- ffmpeg2theora-0.26/SConstruct.ldflags	2010-05-31 23:20:31.000000000 +0200
+++ ffmpeg2theora-0.26/SConstruct	2010-05-31 23:21:53.000000000 +0200
@@ -136,9 +136,9 @@ if not conf.CheckPKG(XIPH_LIBS): 
   Exit(1) 
 ParsePKGConfig(env, XIPH_LIBS)
 
-FFMPEG_LIBS="libavcodec libavformat libavdevice libpostproc libswscale"
+FFMPEG_LIBS="libavcodec libavformat libavdevice libpostproc libswscale libavutil"
 if os.path.exists("./ffmpeg"):
-  os.environ['PKG_CONFIG_PATH'] = "./ffmpeg/libavutil:./ffmpeg/libavformat:./ffmpeg/libavcodec:./ffmpeg/libavdevice:./ffmpeg/libswscale:./ffmpeg/libpostproc:" + os.environ.get('PKG_CONFIG_PATH', '')
+  os.environ['PKG_CONFIG_PATH'] = "./ffmpeg/libavutil:./ffmpeg/libavformat:./ffmpeg/libavcodec:./ffmpeg/libavdevice:./ffmpeg/libswscale:./ffmpeg/libpostproc:./ffmpeg/libavutil" + os.environ.get('PKG_CONFIG_PATH', '')
 if not conf.CheckPKG(FFMPEG_LIBS): 
   print """
       Could not find %s.
