diff -up ffmpeg2theora-0.27/SConstruct.ldflags ffmpeg2theora-0.27/SConstruct
--- ffmpeg2theora-0.27/SConstruct.ldflags	2010-10-14 23:49:39.000000000 +0200
+++ ffmpeg2theora-0.27/SConstruct	2010-10-14 23:49:59.000000000 +0200
@@ -139,7 +139,7 @@ if not conf.CheckPKG(XIPH_LIBS): 
   Exit(1) 
 ParsePKGConfig(env, XIPH_LIBS)
 
-FFMPEG_LIBS=["libavcodec >= 52.30.0", "libavdevice", "libavformat", "libpostproc", "libswscale"]
+FFMPEG_LIBS=["libavcodec >= 52.30.0", "libavdevice", "libavformat", "libpostproc", "libswscale", "libavutil" ]
 if os.path.exists("./ffmpeg"):
   os.environ['PKG_CONFIG_PATH'] = "./ffmpeg/libavutil:./ffmpeg/libavformat:./ffmpeg/libavcodec:./ffmpeg/libavdevice:./ffmpeg/libswscale:./ffmpeg/libpostproc:" + os.environ.get('PKG_CONFIG_PATH', '')
 if not conf.CheckPKG(' '.join(FFMPEG_LIBS)): 
